.github/workflows/lmi-no-code.yml

```diff
@@ -167,17 +169,17 @@ jobs:
         run: |
           rm -rf models
           echo -en "HF_MODEL_ID=s3://djl-llm/llama-2-13b-hf/" > docker_env
-          ./launch_container.sh deepjavalibrary/djl-serving:$DJLSERVING_DOCKER_TAG $PWD/models deepspeed \
+          ./launch_container.sh deepjavalibrary/djl-serving:$DJLSERVING_DOCKER_TAG $PWD/models ${{ matrix.container }} \
           serve
           python3 llm/client.py no_code llama-13b
           docker rm -f $(docker ps -aq)
       - name: Gemma-7b lmi container
-        if: ${{ matrix.container }} == "deepspeed"
+        if: ${{ matrix.container  == 'deepspeed' }}
         working-directory: tests/integration
         run: |
           rm -rf models
           echo -en "HF_MODEL_ID=s3://djl-llm/gemma-7b/" > docker_env
-          ./launch_container.sh deepjavalibrary/djl-serving:$DJLSERVING_DOCKER_TAG $PWD/models deepspeed \
+          ./launch_container.sh deepjavalibrary/djl-serving:$DJLSERVING_DOCKER_TAG $PWD/models ${{ matrix.container }} \
```

Comment:
remove gemma for trtllm

Reply:
This only runs for deepspeed container: ` if: ${{ matrix.container  == 'deepspeed' }}`